package.json

```diff
@@ -101,7 +101,7 @@
     "@inrupt/solid-client": "^1.25.2"
   },
   "engines": {
-    "node": "^14.17.0 || ^16.0.0"
+    "node": "^14.17.0 || ^16.0.0 || ^18.0.0"
```

Comment:
Can we make this ">=14.7.0" so Node 15, 17 & 19 and the upcoming 20 are also listed as having support.

Reply:
We don't support odd releases of NodeJS, which aren't LTS, and we should test Node 20 support before releasing it. I'd prefer releasing with Node 18 support, and check for Node 20 support when it is released in a couple of weeks.